src/js/components/options_selector.js

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react'
+import constants from '../constants'
+
+class OptionsSelector extends React.Component {
+  createOption({ optionName, value, checked }) {
+    return (
+      <label htmlFor={ this.props.name }>
+      <input
+        type='radio'
+        key={ this.props.name + value }
+        value={ value }
+        onChange={ this.props.onChange }
+        checked={ checked } />
+        { optionName }
+      </label>
+    )
+  }
+
+  render () {
+    return (
+      <div className="options-selector">
+        { this.props.name }
+        { this.props.options.map(this.createOption.bind(this)) }
```

Comment:
ah, yeah. Looks like there's a doofy `::` operator in https://babeljs.io/blog/2015/05/14/function-bind/